stake-o-matic/src/main.rs

```diff
@@ -621,6 +621,11 @@ fn main() -> Result<(), Box<dyn error::Error>> {
     let notifier = Notifier::default();
     let rpc_client = RpcClient::new(config.json_rpc_url.clone());
 
+    if !config.dry_run && notifier.is_empty() {
+        error!("A notifier must be active with --confirm");
```

Comment:
Reduces the risk of running with `--confirm` without the discord notification env variable defined